### ISSUE TITLE
Switch to system fonts

### ISSUE THREAD

**Issue by volodymyr-kushnir** [volodymyr-kushnir/recipes#25]
Here's a nice post on the subject — [**Shipping system fonts to GitHub.com**](http://markdotto.com/2018/02/07/github-system-fonts/). Medium ([**System shock**](https://medium.design/system-shock-6b1dc6d6596f#.iw7sldtd6)) and Booking.com ([**Implementing system fonts on Booking.com — A lesson learned**](https://booking.design/implementing-system-fonts-on-booking-com-a-lesson-learned-bdc984df627f)) both have shared their experience as well. In fact, if you're up to know a whole lot more on the matter ― head over to A List Apart's [**Webfonts on the Prairie**](http://alistapart.com/article/webfonts-on-the-prairie) by Richard Fink.

A nice explanation of the term 'system fonts' can also be found in Craig Hockenberry's article [**I Left My System Fonts in San Francisco**](http://furbo.org/2015/07/09/i-left-my-system-fonts-in-san-francisco/) (make sure to also read the follow-up post as it has recent developments on the matter).

Basically, judging from my tests, **Segoe UI** will look almost identical to **Open Sans**, so switching to system fonts will save 8 requests (~100KB). Specific CSS rule can be obtained from the example shared by the guys from WordPress in [**Native Fonts in 4.6**](https://make.wordpress.org/core/2016/07/07/native-fonts-in-4-6/) (they also tested it extensively and they do have a large enough clients base to be a reliable source).
